hw/ip/hmac/data/hmac_testplan.hjson

```diff
@@ -150,6 +150,14 @@
       stage: V3
       tests: ["hmac_smoke"]
     }
+    {
+      name: stress_reset
+      desc: '''Run multiple hmac_long_msg sequences with multiple random on-the-fly resets to ensure
```

Comment:
Nit: I don't think we need the second "multiple"? (Because each sequence will only see one reset)

Reply:
Thanks!
True, I'll fix it.

And from what I saw in my short regression (only this test 50 times), I got 100% FSM coverage, whereas that wasn't always the case with a full regression. Let's see if now we systematically have these transitions covered.